doc/ref_cert/lfn/chapters/chapter11.md

```diff
@@ -5,16 +5,29 @@
 
 ## Table of Contents
 * [11.1 Introduction](#11.1)
-* [11.2 Heading](#11.2)
-* [11.3 Heading](#11.3)
+* [11.2 Openstack Release Comparisons](#11.2)
```

Comment:
can we also add Framework Gaps? for example if there was a gap in Dovetail for Xtesting etc

Reply:
Yes @rabi-abdel .  Added section 11.5 for Framework Gaps.  **Can we Resolve this Conversation?**